dev/functional/tests/cypress/page-objects/student-objects/WelcomeObject.ts

```diff
@@ -21,7 +18,7 @@ export default class WelcomeObject {
   }
 
   loginSIMSText() {
-    cy.contains("Log in to: SIMS - Test");
+    cy.contains("Log in to: SIMS - Dev");
```

Comment:
Does the text need to include the word DEV/TEST? Either way, I believe that this UI will be changing soon. I am not sure why are we investing time on UIs that are probably being changed. Please feel free to elaborate on that.

Reply:
Currently, removed it.